<h3>RNA puzzles</h3>

<p>We develop algorithms to predict the RNA structures of naturally occurring biomolecules and interactions with high accuracy.
  To carry out the RNA structure prediction calculations, we have been developing an automated and efficient program called
  3dRNA (Scientific Reports, 2012). We hypothesize that the organization of RNA structure is largely defined by topological
  constraints, and then build the RNA tertiary structure from the smallest secondary elements (SSEs) by using sequences and
  secondary structure information. The benchmark tests show that 3dRNA can give predictions with reasonable accuracy for
  RNAs of larger size and complex topology. We have continued to improve the 3dRNA automated structure prediction methodology
  by introducing a novel all-heavy-atom knowledge-based statistical potential, 3dRNAscore, to evaluate the predictions and
  to give better accuracy (Nucleic Acids Research, 2015). It turns out that 3dRNAscore performs better than current knowledge-based
  statistical potentials in identifying RNA native structures from a pool of structural decoys as well as ranking a tremendous
  amount of near-native RNA tertiary structures. The 3dRNA methodology is increasingly being used to solve structure-related
  problems in the RNA community.</p>

<h3>Bimolecular mechanisms</h3>

<p>A new role for STAT3 as a regulator of chromatin topology The spatial organization of the genome is known to be important
  for regulation of gene transcription during normal development and in disease states. However, molecular mechanisms governing
  structural rearrangements in the genome are still poorly understood. We proposed a novel mechanism by which phosphorylated
  and unphosphorylated STAT3 and possibly other STAT proteins may regulate gene expression. The anticipated ability of P-STAT3
  tetramers to bend DNA and U-STAT3 dimers to bring together distant DNA regions for DNA looping warrants further investigations
  as a potentially important, and thus far overlooked, mechanism by which STAT3 controls chromatin organization and gene
  expression. The proposed new function of STAT3 also emphasizes an important role of the NDs in STAT signaling. It suggests
  that inhibition of the ND may affect U-STAT dimerization and P-STAT tetramerization, thus preventing/inhibiting DNA looping/bending
  leading to changes in 3D chromatin structure and gene expression. Given the role STAT proteins play in regulation of immune
  responses, inflammation and cancer, the significance of further studies into architectural function of STAT proteins lies
  in discovering novel mechanisms by which STATs may regulate gene expression and designing novel approaches for manipulating
  them for therapeutic applications. (Transcription, 2013)</p>

<p>MD study of the mechanism of unfolding by MID1 Bbox1 domain mutations The zinc-binding Bbox1 domain in protein MID1, a member
  of the TRIM family of proteins, facilitates the ubiquitination of the catalytic subunit of protein phosphatase 2A and alpha4,
  a protein regulator of PP2A. The natural mutation of residue A130 to a valine or threonine disrupts substrate recognition
  and catalysis. While NMR data revealed the A130T mutant Bbox1 domain failed to coordinate both structurally essential zinc
  ions and resulted in an unfolded structure, the unfolding mechanism is unknown. MD simulations can be effective in understanding
  how the two mutations at position 130 perturb the protein structure. The accuracy in computational modeling and MD simulation
  has made it possible to study processes that are too fast to be observed experimentally. MD simulation analysis of the
  two A130T/V mutations revealed that residue A130 served as a hinge point and its mutation caused steric clashes that in
  turn increased the dielectric constant in one of the zinc-binding regions. The increased motion of residues on loop 1 disrupts
  the coordination of both zinc ions that are 13 Ã… apart. (PloS one, 2015)</p>

<h3>Drug design</h3>

<p>Break CDK2/Cyclin E1 Interface Allosterically with Small Peptides Most inhibitors of Cyclin-dependent kinase 2 (CDK2) target
  its ATP-binding pocket. It is difficult, however, to use this pocket to design very specific inhibitors because this catalytic
  pocket is highly conserved in the protein family of CDKs. In this project, we developed some short peptides targeting a
  noncatalytic pocket near the interface of the CDK2/Cyclin complex. Docking and molecular dynamics simulations were used
  to select the peptides, and detailed dynamical network analysis revealed that these peptides weaken the complex formation
  via allosteric interactions. Our Western Blot, Kinase assay and SPR assay experiments showed that upon binding to the noncatalytic
  pocket, these peptides break the CDK2/Cyclin complex partially and diminish its kinase activity in vitro. The binding affinity
  of these peptides measured by Surface Plasmon Resonance is low. (PloS one, 2014)</p>